tasks/install.yml

```diff
@@ -140,6 +206,23 @@
     dest: /etc/tomcat/Catalina/localhost/rest.xml
     backup: yes
 
+- name: config XMLWorflow
+  template:
```

Comment:
These next three config files are examples of ones that we probably want to have in a dspace install repo, rather than in the Ansible role.

Reply:
If this is a general case for all our DSpace instances, we may leave it in the role so we do not have to put it in the specific repos for the DSpace instances.